velodyne_pointcloud/include/velodyne_pointcloud/datacontainerbase.h

```diff
@@ -200,8 +238,11 @@ class DataContainerBase
 
 protected:
   Config config_;
-  boost::shared_ptr<tf::TransformListener> tf_ptr;  ///< transform listener
-  Eigen::Affine3f transformation;
+  boost::shared_ptr<tf2_ros::TransformListener> tf_listener;
+  boost::shared_ptr<tf2_ros::Buffer> tf_buffer;
```

Comment:
Is there a reason to use `boost` instead of `std` here?

Reply:
It was boost before so I used boost again. You already made clear that I should avoid refactoring within this pull request...

Reply:
Apart from this, it would be possible to use std::shared_ptr as the source is compiled with C++11 when I see it correctly.

Reply:
Thanks for the clarification. I'll remove as much `boost` as I can in a future PR.